src/main/java/com/google/sps/servlets/RepListServlet.java

```diff
@@ -5,11 +5,70 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.Scanner;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.util.EntityUtils;
+import org.apache.http.HttpEntity;
+import java.net.URISyntaxException;
+import java.net.URI;
+import org.apache.http.impl.client.HttpClients;
+import com.google.gson.Gson;
```

Comment:
format you imports. They should be in alphabetical order. IDEs and editors probably have a function to do this automatically.

Reply:
alphabetical by the first letter or by the last word (the thing we're actually importing)